contract_api/lambda_handler.py

```diff
@@ -1,93 +1,82 @@
 
-from common.constant import NETWORKS
-from common.repository import Repository
-from common.utils import Utils
-from contract_api.channel import Channel
-from contract_api.search import Search
-from contract_api.service import Service
+from tests.common.constant import NETWORKS
+from tests.common.repository import Repository
+from mpe import MPE
+from registry import Registry
 
-NETWORKS_NAME = dict((NETWORKS[netId]['name'], netId) for netId in NETWORKS.keys())
-obj_util = Utils()
+NETWORKS_NAME = dict((NETWORKS[netId]['name'], netId)
+                     for netId in NETWORKS.keys())
 db = dict((netId, Repository(net_id=netId)) for netId in NETWORKS.keys())
-route_path = {}
 
 
 def request_handler(event, context):
     print(event)
     if 'path' not in event:
-        print("request_handler::path: ", None)
-        return get_response("400", "Bad Request")
+        return get_response(400, "Bad Request")
     try:
         payload_dict = None
+        resp_dta = None
         path = event['path'].lower()
+        stage = event['requestContext']['stage']
+        net_id = NETWORKS_NAME[stage]
+
         if event['httpMethod'] == 'POST':
             body = event['body']
             if body is not None and len(body) > 0:
                 payload_dict = json.loads(body)
-                print("Processing [" + str(path) + "] with body [" + str(body) + "]")
         elif event['httpMethod'] == 'GET':
             payload_dict = event.get('queryStringParameters')
-            print("Processing [" + str(path) + "] with queryStringParameters [" + str(payload_dict) + "]")
-        stage = event['requestContext']['stage']
-        net_id = NETWORKS_NAME[stage]
-        if db[net_id].connection is None:
-            db[net_id] = None
-            db[net_id] = Repository(net_id=net_id)
-
-        data = None
-        if path in ["/service", "/channels", "/fetch-vote", "/user-vote", "/group-info", "/feedback"]:
-            obj_srvc = Service(obj_repo=db[net_id])
-        elif path in ["/available-channels", "/expired-channels"]:
-            obj_chnnl = Channel(net_id, obj_repo=db[net_id])
-        elif path[0:14] in ["/organizations"] or path[0:5] in ["/tags"] or path == "/search":
-            obj_srch = Search(obj_repo=db[net_id])
-
-        if "/service" == path:
-            data = obj_srvc.get_curated_services()
+        else:
+            return get_response(400, "Bad Request")
+
+        if path in ["/service", "/feedback"] or path[0:4] == "/org" or path[0:5] == "/user":
+            obj_reg = Registry(obj_repo=db[net_id])
+
+        if "/org" == path:
+            resp_dta = obj_reg.get_all_org()
+        elif re.match("(\/service)[/]{0,1}$", path):
+            if payload_dict is None:
+                payload_dict = {}
+            resp_dta = obj_reg.get_all_srvcs(qry_param=payload_dict)
+        elif re.match("(\/org\/)[^\/]*(\/service\/)[^\/]*(\/group)[/]{0,1}$", path):
+            params = path.split("/")
```

Comment:
See https://github.com/singnet/snet-marketplace-service/pull/100/files#r299329264 we can split once and also avoid the pattern matching